docs/articles/getting-started.md

```diff
@@ -1,37 +1,37 @@
----
-uid: getting-started
----
-
 # Getting Started
 
-## 1. How to install Colore
+***Important***: *This page is for v6.0.0 of Colore, which is currently only available as a pre-release. For the v5.2 getting started guide (not recommended), [[click here|Getting-started-(v5.2)]].*
+
 [![NuGet version][ngverbadge]][ng]
 [![MyGet version][mgverbadge]][mg]
 
-The easiest way to include Colore into your project is to right click on your C# Project in the Visual Studio Solution Explorer and Choose "Manage NuGet Packages..."
+## 1. Installing Chroma SDK
+Razer Chroma SDK is a C++-tool which allows us to access Chroma devices. It is the backend (or native/unmanaged code) behind Colore. It is expected to be automatically installed with Synapse once you plug in a Chroma device, but in some cases that might not work (see [#263](https://github.com/chroma-sdk/Colore/issues/263#)). In this case you can download the so called [Chroma SDK Core](https://assets.razerzone.com/dev_portal/downloads/Razer_Chroma_SDK_Core_v1.10.6.exe) which is just the SDK without Synapse.
 
-From there you can search online for Packages.
-The NuGet Package manager sadly looks way different depending on the Version of Visual Studio but it should be easy to find Colore.
+## 2. How to install Colore 
+The easiest way include Colore into your project is to right click on your C# Project in the Visual Studio Solution Explorer and Choose "Manage NuGet Packages..."
+
+From there you can search online for Packages. **Be sure to enable Pre-releases** or you will not be able to download v.6.00.
 Just search for "Colore" and install it.
-After that you should see a Reference being added to "Colore".
+After that you should see a reference being added to "Colore".
 
-### Pre-release versions
-If you want to test the absolute latest features in Colore, which may not be fully ready for production use yet, you can install pre-release Colore packages from [our MyGet feed][mg]. You can either install a version manually with the commands listed on the page, or add the feed to your NuGet settings: `https://www.myget.org/F/chroma-sdk/api/v3/index.json`.
+### Other Pre-release versions
+If you want to test the absolute latest features in Colore, which may not be fully ready for production use yet, you can install pre-release Colore packages from [our MyGet feed][mg]. You can either install a version manually with the commands listed on the page, or add the feed to your NuGet settings: `https://www.myget.org/F/coralestudios/api/v3/index.json`.
```

Comment:
`coralestudios` is the old MyGet feed, `chroma-sdk` is the new one, so it should still be the same URL.